src/main/scala/nl.knaw.dans.easy.deposit/docs/DatasetMetadata.scala

```diff
@@ -78,8 +78,9 @@ case class DatasetMetadata(private val identifiers: Option[Seq[SchemedValue]] =
   atMostOne(datesCreated)
   atMostOne(datesAvailable)
 
+  private lazy val authors: Seq[Author] = (contributors.toSeq ++ creators.toSeq).flatten
   // duplicates for plain strings in dcterms (which has no place for IDs as contributors/creators do)
-  lazy val rightsHolders: Seq[String] = (contributors.toSeq ++ creators.toSeq).flatten
+  lazy val rightsHolders: Seq[String] = authors
     .withFilter(_.isRightsHolder)
```

Comment:
`withFilter` + `map` == `collect`

Reply:
`.collect { case author if author.isRightsHolder => author.toString }`

makes it harder for me to tell the forest from the trees